README.md

```diff
@@ -78,47 +78,229 @@ Here's what each of these steps does:
-  allocate a 1 TB partition and check out cargobomb there.
-- I run in a screen session with 4 windows: "master", "tc1",
-  "tc2", "upload"
-- Create a directory to hold reports called `cargobomb-reports`. The
-  "upload" screen is in this directory, all others are in `cargobomb`.
-- In "master" run `cargo run -- prepare-local` once a week or so,
-  between runs
-- In "master" run `cargo run -- define-ex --ex nightly-2017-04-24`.
-  The experiment name is named after the channel being compared to stable,
-  coupled with the current date. This experiment name will correspond
-  to the directory name containing the final report.
-- In "master" run `cargo run -- prepare-ex --ex nightly-2017-04-24`
-- In "tc1" run `cargo run -- run-tc --ex nightly-2017-04-24 stable`
-- At the same time, in "tc2" run `cargo run -- run-tc --ex nightly-2017-04-24 nightly`
-- That will take about 4 days. The runs can be stopped and restarted
-  at any time.
-- In "master" run `cargo run -- publish-report --ex nightly-2017-04-24 s3://cargobomb-reports/nightly-2017-04-24`
-- In "master" run `cargo run -- delete-all-target-dirs --ex nightly-2017-04-24`
-  to free up disk space
-
-Now the report is uploaded to http://cargobomb-reports.s3-website-us-west-1.amazonaws.com/nightly-2017-04-24/index.html
+Toolchains for rust PRs that have been built by by asking bors to try a PR can
+be specified using `try#<SHA1 of try merge>`. You will probably want to specify
+the comparison commit as `master#<SHA1 of master before try merge>`.
+
+## Operational workflow for production cargobomb
+
+### Getting access
+
+There are three 'official' cargobomb machines:
+
+ - cargobomb-test (54.177.234.51) - 1 core, 4GB RAM, for experimenting
+ - cargobomb-try (54.241.86.211) - 8 core, 30GB RAM, for doing PR runs
+ - cargobomb-prod (54.177.126.219) - 8 core, 30GB RAM, for doing beta runs (but can do PR runs if free)
+
+These can only be accessed via the bastion (you `ssh` to the bastion,
+then `ssh` to the cargobomb machine). The bastion has restricted access
+and you will need a static IP address (if you have a long-running server
+in the cloud, that's usually fine) and a public SSH key (you should add
+the key to github and then link to https://github.com/yourusername.keys,
+once you have access to the bastion you can manage your own keys).
+
+With these two pieces of information in hand, ask acrichto or brson to
+add you to the bastion and all three machines and they'll let you know
+the bastion IP. You can now either edit your `~/.ssh/config` on your
+static IP machine to contain
+
+```
+Host rust-bastion
+    # Bastion IP below
+    HostName 0.0.0.0
+    User bastionusername
+Host cargobomb-test
+    HostName 54.177.234.51
+    ProxyCommand ssh -q rust-bastion nc -q0 %h 22
+    User ec2-user
+# [...and so on for cargobomb-try and cargobomb-prod...]
+```
+
+which will let you do `ssh cargobomb-test` etc from your static IP
+machine, or you can use ssh-agent forwarding with `ssh -A`.
```

Comment:
Gah, I forgot a "**not**" there. I don't think we should mention ssh-agent forwarding at all.